pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -20,160 +20,105 @@ import (
 	"reflect"
 	"testing"
 
-	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
 )
 
 const (
+	scoreWithNormalizePlugin1  = "score-with-normalize-plugin-1"
+	scoreWithNormalizePlugin2  = "score-with-normalize-plugin-2"
 	scorePlugin1               = "score-plugin-1"
-	scorePlugin2               = "score-plugin-2"
-	scorePlugin3               = "score-plugin-3"
 	pluginNotImplementingScore = "plugin-not-implementing-score"
-	weight1                    = 2
-	weight2                    = 3
-	weight3                    = 4
 )
 
-// TestScorePlugin1 and 2 implements ScoreWithNormalizePlugin interface,
-// TestScorePlugin3 only implements ScorePlugin interface.
-var _ = ScoreWithNormalizePlugin(&TestScorePlugin1{})
-var _ = ScoreWithNormalizePlugin(&TestScorePlugin2{})
-var _ = ScorePlugin(&TestScorePlugin3{})
+// ScoreWithNormalizePlugin1 and 2 implements ScoreWithNormalizePlugin interface,
+// ScorePlugin1 only implements ScorePlugin interface.
+var _ = ScoreWithNormalizePlugin(&ScoreWithNormalizePlugin1{})
+var _ = ScoreWithNormalizePlugin(&ScoreWithNormalizePlugin2{})
+var _ = ScorePlugin(&ScorePlugin1{})
 
-type TestScorePlugin1 struct {
-	// If fail is true, NormalizeScore will return error status.
-	fail bool
-}
-
-// NewScorePlugin1 is the factory for NormalizeScore plugin 1.
-func NewScorePlugin1(_ *runtime.Unknown, _ FrameworkHandle) (Plugin, error) {
-	return &TestScorePlugin1{}, nil
-}
+func NewScoreWithNormalizePlugin1(inj injectedResult) Plugin { return &ScoreWithNormalizePlugin1{inj} }
```

Comment:
s/NewScoreWithNormalizePlugin1/newScoreWithNormalizePlugin1/, no need to export them.